### PR TITLE
Add missing hackathon groups

### PR DESCRIPTION
**Pull request by JoseEspinosa**
@netlify /events/2024/hackathon-barcelona

### CODE REVIEW

**Comment by JoseEspinosa** [comment]
nf-core-bot fix linting

**Comment by JoseEspinosa** [comment]
@nf-core-bot fix linting

**Comment by JoseEspinosa** [comment]
@ewels @FranBonath added the missing hackathon projects here but `differentialabundance` that has been added already [here](https://github.com/nf-core/website/pull/2798)

**Comment by FranBonath** [comment]
also, Kubras github handle and name are the wrong way round. Like this the pic cant be found.

File: sites/main-site/src/content/events/2024/hackathon-barcelona.mdx [hunk 1]
```diff
@@ -243,6 +243,44 @@ You can move freely between projects throughout the event.
 
 </HackathonGroup>
 
+<HackathonGroup
+    title="nf-core/variantbenchmarking"
+    category="pipelines"
+    leaders={{
+        'Kübra Narcı': 'kubranarci'
```
Comment (FranBonath) [inline sites/main-site/src/content/events/2024/hackathon-barcelona.mdx:250]:
```suggestion
        'kubranarci': 'Kübra Narcı'
```

**Comment by FranBonath** [comment]
still give this one an approval so we can get this merged.

**Comment by JoseEspinosa** [comment]
> also, Kubras github handle and name are the wrong way round. Like this the pic cant be found.

Good spot, thanks!

**Pull request merged**